docs/modules/ROOT/pages/mathematical-operators.adoc

```diff
@@ -0,0 +1,93 @@
+[[mathematical-operators]]
+= Mathematical operators
+
+Mathematical operators are a handy way to update numerical fields in a single roundtrip to the DB.
+
+Specific operators are available on different numerical types (`Int`, `Float`, xref::type-definitions/types.adoc#type-definitions-types-bigint[`BigInt`]).
+
+We chose to keep operators separated by types as some operators are not available to Natural numbers.
+
+== Supported locations
+Mathematical operators are supported in Update Mutations within these entities:
+
+* `Node`
+* `Nested Nodes`
+* `Relationship properties`
+* `Interfaces`
+
+== Int and BigInt operators
+For `Int` and `BigInt` types, the following operators are available:
+
+* `_DECREMENT`
+* `_INCREMENT`
+
+== Float operators
+For `Float` type, the following operators are available:
+
+* `_ADD`
+* `_SUBTRACT`
+* `_MULTIPLY`
+* `_DIVIDE`
+
+== The social platform schema
```

Comment:
Is this an example?

Maybe we should call this section `example`?

Reply:
It was intended more like a shared scenario to be used in the followed examples like the "Simple increment operation", maybe we could change it to scenario?
```suggestion
== The social platform scenario
```

Reply:
I'm happy with this being a section, but it should be inside an "Examples" section as it is confusing whether this is part of the examples or documentation

Reply:
I see what you mean, I agree, it makes sense to me to move this to the "Examples" section 👍